SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialReceiver.h

```diff
@@ -203,6 +203,39 @@ class USpatialReceiver : public UObject
 
 	void PeriodicallyProcessIncomingRPCs();
 
+	bool NeedToLoadClass(const FString& ClassPath);
```

Comment:
Can this new code be moved to a separate class?

Reply:
If we don't end up moving this can we make it a static function?